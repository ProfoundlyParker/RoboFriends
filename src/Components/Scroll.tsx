import React, { ReactNode } from "react";

// ReactNode represents any node that could be rendered in React (ex. JSX, strings, numbers)
type ScrollProps = {
    children: ReactNode;
}

//Scroll component that handles Y overflow
const Scroll: React.FC<ScrollProps> = (props) => {
    return (
        <div className='overflow-y-scroll border-none h-scroll-h'>
            {props.children}
        </div>
    );
};

export default Scroll;