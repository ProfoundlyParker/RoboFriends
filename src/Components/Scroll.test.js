import { render } from "@testing-library/react";
import Scroll from "./Scroll";

//Snapshot
it("expect to snapshot Scroll component", () => {
  const { asFragment } = render(<Scroll />);
  expect(asFragment()).toMatchSnapshot();
});