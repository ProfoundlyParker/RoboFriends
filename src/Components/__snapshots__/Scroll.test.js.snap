// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to snapshot Scroll component 1`] = `
<div
  style={
    Object {
      "border": "none",
      "height": "500px",
      "overflowY": "scroll",
    }
  }
/>
`;
